### ISSUE TITLE
Alexa Typ "Rolladen"

### ISSUE THREAD

**Issue by Marsx79** [ioBroker/ioBroker.iot#82]
Da Alexa jetzt die Smarten Rollos/Jalousien von Ikea unterstützt, werden jetzt wohl auch die Befehle "öffnen, schließen, hoch, runter usw" unterstützt. Ist es geplant einen Typ "Rolladen" oder wie auch immer der dann heißen mag einzuführen?

**Comment by Apollon77** [comment]
Ich fürchte geht nur mit Skill-v3 ...

**Comment by udochrist** [comment]
> Ich fürchte geht nur mit Skill-v3 ...

Stimmt. Dann sollte an diesen Feature - Request aber auch mal ein Meilenstein gehängt werden. Ich hätte den Support nämlich ebenfalls sehr gerne dafür...

**Comment by Apollon77** [comment]
Ich kann gern einen Meilenstein „v3 skill“ erstellen und an das issue hängen. Das bringt dann genau was?

Oder erwartest du damit auch einen Termin? ;-) ein Termin hängt davon ab wann mal jemand Zeit für dieses Thema hat. Der v3 support wird einiges an Aufwand kosten. 

Das ist die aktuelle Realität.

**Comment by zaunermax** [comment]
Ich würde das auch eventuell in meiner Freizeit angehen, jedoch habe ich auch egentlich für diesen Skill "bezahlt", also sollte das eigentlich schon von den Empfängern dieses Gelds implementiert werden? Oder liege ich da falsch?

**Comment by Apollon77** [comment]
IoBroker ist Open Source. Was du bezahlst kann man jetzt so oder so sehen ... formal bezahlst du dafür das ioBroker jeden request der per Alexa bzw. Google gemacht wird und wo Amazon Geld für verlangt auch bezahlen kann. So funktioniert die Cloud Infrastruktur hinten dran. Jeder request, Traffic und die Rechenleistung wird abgerechnet und ein skill muss Systembedingt cloudbasiert sein. Mit v3 kommt Push von state Werten dazu was mehr requests bedeuten wird. 

Die v3 Implementierung ist zu großen Teilen (90% ca) im iot Adapter. Der ist Open Source.

Vllt hilft dadurch deine Frage selbst zu beantworten. 

Falls du Interesse hast mitzuarbeiten melde dich gern nochmals.

**Comment by zaunermax** [comment]
@Apollon77 ich würde mich beteiligen, wenn du mir irgendwo kurz sagen könntest, wie ich das am besten testen kann bzw developen kann? 

Bei der v3 implementierung könnte man ja vorerst mal das state reporting weglassen, um die implementierung einfacher zu machen...

**Comment by Apollon77** [comment]
Ich glaube state reporting ist teilweise Pflicht. 

Ich bespreche  mich demnächst mit bluefox dazu, dann komme ich gern auf dein Angebot zurück. Ich denke die Typen kann man gut aufteilen wenn die Basis definiert ist.

**Comment by zaunermax** [comment]
melde dich bei mir, wenn du was hast was ich vllt implementieren könnte

**Comment by Marsx79** [comment]
@Apollon77 @zaunermax  Ich wollte mal nachfragen ob an der V3 Implementierung schon gearbeitet wird? Ich weiß das das viel Arbeit ist (vielen Dank an dieser Stelle für die tolle Arbeit!) oder ob das in naher oder ferner :-) Zukunft geplant ist?

**Comment by Apollon77** [comment]
Gestartet noch nicht. Ich werde so wie ich es die Tage schaffe emal zusammenschreiben was der "plan" wäre und wo wir hin wollen, damit die Gerätevielfalt der Assistenzsysteme nicht zuviel Dev Aufwand frisst. Wir müssen hier das ganze etwas generischer Abbilden können.

**Comment by drohne200** [comment]
Wie siehts aus mit dem Typ Rolladen? ;-)

**Comment by Apollon77** [comment]
So, hier geht es los mit dem generellen Thema ... we start here ... https://forum.iobroker.net/topic/35020/devices-alias-assistenten-visualisierungen-die-zukunft/8

**Comment by Buhu91** [comment]
Hat sich hier noch was getan? Inzwischen gibt es in der Alexa App eine Vielzahl von Smarttypes. Aber auswählen kann ich im Adapter immer noch nur eine Hand voll. Oder gibt es hier einen anderen Weg?

**Comment by Apollon77** [comment]
geht am Ende alles auf "v3 Skill support" zurück. Ist geplant

**Comment by Buhu91** [comment]
Danke für die Antwort, bis wann ist dann der release geplant? Bin leider Aktuell etwas draußen mit den aktuellsten Entwicklungen über das Forum.

**Comment by Apollon77** [comment]
Das hängt von der verfügbaren zeit und anderer Themen ab

**Comment by Marsx79** [comment]
Gibt es schon Neuigkeiten zum v3 Skill support?

**Comment by Apollon77** [comment]
Keine ausser" soon" ... aber kommt näher das ich zeit finde daran zu arbeiten

**Comment by theengineer0815a** [comment]
Ich häng mich auch mal ran!
"Blinds" vermisse ich auch schmerzhaft.
